Dockerfile

```diff
@@ -9,51 +9,35 @@ ENV LC_ALL     en_AU.UTF-8
 
 # Use nearby apt mirror
 COPY ./files/sources.list /etc/apt/sources.list
-RUN sed -i.bak "s/<mirror>/http:\/\/mirror.internode.on.net\/pub\/ubuntu\/ubuntu/g" /etc/apt/sources.list
-RUN sed -i.bak "s/<version>/$(sed -n "s/^.*CODENAME=\(.*\)/\1/p" /etc/lsb-release)/g" /etc/apt/sources.list
-
-# Start by adding/updating required software
-RUN apt-get update
-RUN apt-get -y install apt-transport-https ca-certificates
-RUN apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys 58118E89F3A912897C070ADBF76221572C52609D
-RUN echo "deb https://apt.dockerproject.org/repo ubuntu-wily main" > /etc/apt/sources.list.d/docker.list
-
-# Update again now with the extra repos
-RUN apt-get update
-RUN apt-get -y dist-upgrade
-
-# Install new packages we need
-RUN apt-get -y --force-yes install docker-engine
-
-# Configure timezone
-RUN echo "Australia/Adelaide" > /etc/timezone; dpkg-reconfigure tzdata
```

Comment:
I'm talking about `echo "Australia/Adelaide" > /etc/timezone; dpkg-reconfigure tzdata`. We're only setting php timezone in the MR.